pkg/utils/admission/cleanup.go

```diff
@@ -37,3 +37,48 @@ func GetCleanupPolicies(request *admissionv1.AdmissionRequest) (kyvernov1alpha1.
 	}
 	return policy, emptypolicy, nil
 }
+
+func FetchUniqueKinds(polspec kyvernov1alpha1.CleanupPolicySpec) []string {
+	var kindlist []string
```

Comment:
Use a set

Reply:
> Do we need changes in the clusterRole resource for providing Kyverno Service Account permission to delete particular kinds in all namespaces?

Ultimately this is an end user decision. We can change it for the tests but it will have to be configurable in the helm chart in the end.